client/src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -26,15 +28,15 @@ const LanguageSelector = (props) => {
 
     return (
 
-        <div className = {`language-selector ${props.size ? props.size : ''}`} onClick = {handleSliderClick}>
+        <div className = {`language-selector ${size ? size : ''}`} onClick = {handleSliderClick}>
```

Comment:
The ternary in this line is unnecessary, because if size is undefined, the div won't get any class beside `"language-selector"`.
```suggestion
        <div className = {`language-selector ${size}`} onClick = {handleSliderClick}>
```

Reply:
Fixed in: 3fae083